src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package kitchenpos.application;
+
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.fake.menuGroup.TestMenuGroupRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class MenuGroupServiceTest {
+    @Test
+    @DisplayName("메뉴그룹을 생성할 수 있다")
+    void create_menu_group() {
+        MenuGroupService menuGroupService = new MenuGroupService(new TestMenuGroupRepository());
+        String menuGroupName = "TestMenuGroup";
+        MenuGroup request = new MenuGroup();
+        request.setName(menuGroupName);
+        MenuGroup menuGroup = menuGroupService.create(request);
+
+        assertAll(
+                () -> assertThat(menuGroup.getId()).isNotNull(),
+                () -> assertThat(menuGroup.getName()).isEqualTo(menuGroupName)
+        );
+    }
+
+    @ParameterizedTest
+    @DisplayName("메뉴그룹이름은 존재해야하며 길이가 1 이상이다")
+    @NullSource
```

Comment:
여기도 빈문자열 체크를 포함하면 좋을 것 같아요
```suggestion
    @NullAndEmptySource
```

Reply:
넵 NullAndEmptySource 로 변경했습니다